src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Todo } from '../../interfaces/Todo';
+import { TodoItem } from '../TodoItem';
+
+interface Props {
+  items: Todo[];
+  toggleAll: () => void;
+}
+
+export const TodoList: React.FC<Props> = ({ items, toggleAll }) => (
+  <section className="main">
+    <input
+      type="checkbox"
+      id="toggle-all"
+      className="toggle-all"
+      data-cy="toggleAll"
+      onChange={toggleAll}
+    />
+    <label htmlFor="toggle-all">Mark all as complete</label>
+
+    <ul className="todo-list" data-cy="todoList">
```

Comment:
```suggestion
    <ul className="todo-list" data-cy="todosList">
```

Reply:
It may be a reason
<img width="912" alt="image" src="https://github.com/mate-academy/react_todo-app/assets/98764641/5bd6ac78-598f-4aa2-9c7c-7500118aaaee">